src/scripts/components/stories/story-gallery/story-gallery.tsx

```diff
@@ -118,7 +122,14 @@ const StoryMedia: FunctionComponent<Props> = ({
                 key={index}
                 style={{width: `${imageWidth}%`}}>
                 <div className={styles.imageContainer}>
-                  <img className={styles.photo} src={imageUrl} />
+                  <img
```

Comment:
The story gallery component is getting really big. Could you clean it up by pulling out a StoryGalleryImage component with captions for example

Reply:
aye 👌  will do